### PR TITLE
Add `best` command

### PR DESCRIPTION
**Pull request by indiv0**
@ultrabear not sure how to check if the types are correct and how to reformat the file. What are the commands for that?

### CODE REVIEW

File: ferris_elf/__init__.py [hunk 1]
```diff
@@ -314,6 +314,43 @@ async def formatted_scores_for(
...
+        # if the aoc command was sent in a guild that isnt the guild of the user we have here, then using <@id>
+        # will render as <@id>, instead of as @person, so we have to fallback to using the name directly
+        if guild is None or guild.get_member(user) is None:
+            userobj = bot.get_user(user) or await bot.fetch_user(user)
+            if userobj:
+                builder.write(
+                    f"\td{opt_day}: {escape_markdown(userobj.name)} - **{ns(opt_bench_time)}**\n"
+                )
+            continue
+        builder.write(f"\td{opt_day}: <@{user}> - **{ns(opt_bench_time)}**\n")
```
Comment (indiv0) [inline ferris_elf/__init__.py:347]: @bend-n as you are the formatting expert, any ideas on how to prettify the output?

image

**Comment by ultrabear** [comment]
pyright can be run with `uv run pyright`
ruff linting can be run with `uv run ruff check` and formatting with `uv run ruff format`

**Comment by ultrabear** [comment]
you can also use `uv venv` to get shell specific instructions to enter the venv, then you dont need to prefix commands with uv run (just remember to exit the venv when you are done)

**Comment by indiv0** [comment]
damn nixOS:
```
$ uv run ruff check
Could not start dynamically linked executable: ruff
NixOS cannot run dynamically linked executables intended for generic
linux environments out of the box. For more information, see:
https://nix.dev/permalink/stub-ld
```

**Comment by indiv0** [comment]
Nevermind

File: ferris_elf/__init__.py (hunk 1 above)
Comment (bend-n) [reply ferris_elf/__init__.py:351]: lol… i was just looking at this and cringing at the `-` while waiting for the notification to load.
do you think we could have it more like this?
image
could also keep it split, and just merge in the duplicates, such as (keeping it more like `aoc`'s result)
image
possibly removing the duplication of the day
image

Comment (indiv0) [reply ferris_elf/__init__.py:351]: this would be great

**Pull request merged**